packages/calypso-products/src/plans-list.tsx

```diff
@@ -1588,7 +1588,16 @@ const getPlanJetpackSecurityT1Details = (): IncompleteJetpackPlan => ( {
 		),
 	getFeaturedDescription: () =>
 		translate(
-			'Easy-to-use, comprehensive WordPress site security including backups, malware scanning, and spam protection.'
+			'This bundle includes:{{ul}}{{li}}VaultPress Backup{{/li}}{{li}}Scan{{/li}}{{li}}Akismet Anti-spam{{/li}}{{/ul}}',
+			{
+				components: {
+					ul: <ul />,
+					li: <li />,
+					br: <br />,
+				},
+				comment:
+					'{{ul}}{{ul/}} represents an unorder list, {{li}}{/li} represent an item and {{br/}} represents a line break',
```

Comment:
```suggestion
					'{{ul}}{{ul/}} represents an unorder list, {{li}}{/li} represent a list item and {{br/}} represents a line break',
```

Minor improvement suggestion 😄

Reply:
Thanks, I fixed it 👍